musync_save/main_window.py

```
@@ -0,0 +1,784 @@
+
+    def CheckUpdate(self) -> None:
+        """检查软件更新"""
+        def CheckVersion(local:list[int], target:list[int], channel:bool=False)->bool:
+            """版本号比较
+            param:
+                local: list[int] - 本地版本号列表
+                target: list[int] - 目标版本号列表
+                channel: bool - 更新通道 (is pre-release)
+            return: bool - 是否存在更新
+            """
+            if (local[0] > target[0]): return False
```

Ruff (0.15.9)

[error] Multiple statements on one line (colon)

(E701)